frontend/static/main.js

```diff
@@ -233,18 +233,46 @@ app.controller('Stats', ['$scope', '$http', '$routeParams', '$timeout', '$q', '$
 
         $http.get('/api/stats/', options).then(function(response) {
 
-            var series = _.map(_.groupBy(response.data, "name"), function(data, name) {
-                return {
+            var series = [];
+            var i = 0;
+            _.each(_.groupBy(response.data, "name"), function(data, name) {
+
+                // data itself
+                series.push({
                     name: name,
+                    color: Highcharts.getOptions().colors[i],
+                    zIndex: 1,
                     data: _.map(data, function(data) {
                         return {
                             x: Date.parse(data.created_at),
                             y: data.measurement,
+                            stdev: data.stdev,
                             result_id: data.result,
                             build_id: data.build_id
                         };
                     })
-                };
+                });
+
+                // range of values, based on the standard deviation
+                series.push({
+                    name: name + ' stdev',
+                    type: 'arearange',
+                    color: Highcharts.getOptions().colors[i],
+                    lineWidth: 0,
+                    linkedTo: ':previous',
+                    fillOpacity: 0.3,
+                    zIndex: 0,
+                    data: _.map(data, function(point) {
+                        return [
+                            Date.parse(point.created_at),
+                            // 99.73% of values are in the range (mean ± 3 * stdev)
+                            point.measurement - 3 * point.stdev,
```

Comment:
Well, statistically just 1 stdev it not very representative of the the actual values. i.e, the range (mean ± stdev) will only contain ~ 68% of the values in the sample. If you think 3 is too much, we can use 2, but 1 is definitively wrong.

See https://en.wikipedia.org/wiki/68%E2%80%9395%E2%80%9399.7_rule